scripts/setup_wasi.sh

```diff
@@ -67,4 +67,13 @@ if [ ! -d $WASI_SDK_INSTALL_DIR ]; then
 
     echo "Extracting to $WASI_SDK_INSTALL_DIR..."
     tar zxf $OUTFILE -C $TOOL_INSTALL_DIR
+    
+    # Rename the architecture-specific directory to a generic name
+    EXTRACTED_DIR="$TOOL_INSTALL_DIR/wasi-sdk-$WASI_SDK_VERSION_FULL-$SYS_ARCH-$SYS_OS"
+    GENERIC_DIR="$TOOL_INSTALL_DIR/wasi-sdk-$WASI_SDK_VERSION_FULL"
+    
+    if [ -d "$EXTRACTED_DIR" ]; then
+        echo "Renaming $EXTRACTED_DIR to $GENERIC_DIR..."
+        mv "$EXTRACTED_DIR" "$GENERIC_DIR"
+    fi
```

Comment:
I think I should remove this check because if `WASI_SDK_TARBALL` changes again in the future, it will bypass this check and cause a hard-to-debug error during the build. What do you guys think?

Reply:
Makes sense 👍